cypress/support/cypress-commands/assertion.js

```diff
@@ -175,8 +175,15 @@ Cypress.Commands.add(
                 ? (isNullCheckSkipped = true)
                 : (isNullCheckSkipped = false);
             }
-            // Checking if the error is null in the response and if the error is expected or not.
-            if (!UTILS.getEnvVariable(CONSTANTS.IS_SCENARIO_EXEMPTED, false)) {
+            // Skip the null check for exempted scenarios based on the error response.
+            if (UTILS.getEnvVariable(CONSTANTS.IS_SCENARIO_EXEMPTED, false)) {
+              // Performing error null check validation if the response error is not null.
+              if (response?.error == null) {
+                cy.errorNullCheck(response, CONSTANTS.ERROR).then((result) => {
+                  validationCheck.push(result);
+                });
+              }
+            } else {
```

Comment:
why we need to skip this check if it's an error?
Can't we do similar to negative cases where we are expecting error ? 

[8/18/2024 6:45:05 AM][loggingValidationCheckResult][Error null Check: PASS. Expected error in response: "entityId" is a required property ]

Reply:
addressed the comment by adding this line